### DISCUSSION TITLE
Incorrect email validation in signup form

### DISCUSSION THREAD

**Discussion by amkelly** [community/community#109158] [GitHub Education] [answered]
### Hi everyone,

I'm looking for a contact for the global campus that can change an incorrect setting on the form. It's trying to validate campus emails against a subdomain:
We require applicants of Paul Smith's College to use one of these school-issued email addresses to apply:
- s.paulsmiths.edu

We only used the s.paulsmiths.edu subdomain emails for students a year or two maybe 4 or 5 years ago. I've validated my faculty email at ak****@paulsmiths.edu previously and need to revalidate it. I don't see a way to get around this incorrect assumption about email subdomain.

**Comment by namith1003** [comment, marked as answer]
according to the GitHub maintainer in the discussion

https://github.com/orgs/community/discussions/53069

this issue happens quite often and the best thing you can do as he says in his reply is the following

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hopefully, this answers your question, feel free to contact me if there is anything else,

A.A Namith Nimlaka
